quantum/process_keycode/process_sequencer.c

```diff
@@ -0,0 +1,76 @@
+#include "process_sequencer.h"
+
+#if defined(SEQUENCER_ENABLE)
+
+bool process_sequencer(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case SQ_ON:
+            if (record->event.pressed) {
```

Comment:
Move this if statement outside the switch block, to save a bit on code size.

Reply:
I have considered that, but thought I should still return `false` no matter the value of `record->event.pressed`, to exit `process_record_quantum` as fast as possible; since the keycode belongs to the `sequencer` domain. Is that unnecessary optimization? If code size is the only concern, I could maybe reduce it with a macro.

Reply:
Or I could introduce a `SEQUENCER_KEYCODE_MIN` and `SEQUENCER_KEYCODE_MAX` aliases and return `false` for any keycode in this range.

Reply:
The `return false`s aren't inside the if statements. See the backlight keycode handling, for example: https://github.com/qmk/qmk_firmware/blob/master/quantum/process_keycode/process_backlight.c
You can see there are a lot less branches in there, just the initial keydown check and the switch cases.